.github/workflows/cypress.yml

```
@@ -18,6 +18,10 @@ jobs:
     strategy:
       matrix:
         browsers: [chrome, firefox]
+        # This should spin up machines which Cypress can use
+        # to run tests in parallel
+        machines: [1, 2, 3]
+        fail-fast: false
```

```suggestion
      fail-fast: false
      matrix:
        browsers: [chrome, firefox]
        # This should spin up machines which Cypress can use
        # to run tests in parallel
        machines: [1, 2, 3]
```
Both `fail-fast` and `matrix` are options of `strategy`.